vm/src/obj/objtuple.rs

```diff
@@ -213,18 +213,22 @@ fn tuple_new(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 fn tuple_repr(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     arg_check!(vm, args, required = [(zelf, Some(vm.ctx.tuple_type()))]);
 
-    let elements = get_elements(zelf);
+    let s = if let Some(_guard) = ReprGuard::enter(zelf) {
```

Comment:
Is it possible to create a recursive tuple?

Reply:
Yes, tuple elements are still mutable so putting the tuple into one if it's elements. makes it recursive